apps/platform-app/src/app/[locale]/(profile)/_actions/mutations/saveMyProfile.ts

```diff
@@ -20,6 +20,7 @@ const profilePendingFields: EditProfileFormFields[] = [
   'fullName',
   'email',
   'avatarUrl',
+  // 'slug',
```

Comment:
**nit:** Comment to remove?

Reply:
this is added to remember to discuss if changed slug should change profile state to PENDING or not